README.md

```diff
@@ -1,48 +1,79 @@
+| `32...`  | 500 (Internal Server Error) | -                           | -                                        |
+| `33...`  | 503 (Service Unavailable)   | -                           | -                                        |
+| `41...`  | `Test Tech Ltd`             | `9999999999`                | 404 (Not Found)                          |
+| `421...` | `Test Tech Ltd`             | `9299999998`                | 400 (Bad Request) invalid CTUTR          |
+| `422...` | `Test Tech Ltd`             | `9289999996`                | 400 (Bad Request) invalid start date     |
+| `423...` | `Test Tech Ltd`             | `9279999994`                | 400 (Bad Request) invalid end date       |
+| `424...` | `Test Tech Ltd`             | `9269999992`                | 400 (Bad Request) invalid correlation ID |
+| `43...`  | `Test Tech Ltd`             | `9399999995`                | 422 (Unprocessable Entity)               |
+| `44...`  | `Test Tech Ltd`             | `9499999992`                | 500 (Internal Server Error)              |
+| `46...`  | `Test Tech Ltd`             | `9699999997`                | 502 (Bad Gateway)                        |
+| `47...`  | `Test Tech Ltd`             | `9799999994`                | 503 (Service Unavailable)                |
 
 ## Individual Data
 
-| NINO          | Date of Birth | SA UTR (Citizens Details) | SA status                   |
-| ------------- | ------------- | ------------------------- | --------------------------- |
-| `NS...`       | `01-12-1922`  | -                         |  -                          |
-| anything else | `01-12-1922`  | `1234567895`              | return found                | 
-| -             | -             | `1111...`                 | no return found             | 
-| -             | -             | `2222...`                 | notice to file issued       | 
-| -             | -             | `3333...`                 | 500 (Internal Server Error) | 
-| -             | -             | anything else             | return found                |      
+The table below describes responses to a number of API's. The journeys for individuals that you will be able to trigger on the applicant
+frontend service will be rooted in the national insurance number (NINO) and the self assessment unique tax reference 
+(SA UTR) if an SA journey is chosen. The columns should be interpreted like so:
+- The `Date of Birth` and `SA UTR` columns are the values that will be returned by the citizen details API stub for the
+  given NINO. The SA UTR is an optional field in the response and it is not always returned by the stub. The name that is
+  returned by the stub is always the same
+- the `SA status` column describes the stub responses to the SA return status API. The input to this API is an SA UTR 
+
+If using a stubbed login, to be able to use an SA UTR which is different thn `1234567895` use an `NS...` NINO in the 
```

Comment:
reword then,

Reply:
updated - any better?

Reply:
i see 'thn' , please change 'then'

Reply:
whoops 😆  I meant `than` but apparently it's more grammatically correct to use `from` according to intellij

Reply:
nice